docs/staging.asciidoc

```diff
@@ -8,9 +8,8 @@ Introduction
 ------------
 [id="intro"]
 
-The staging plugin is used to enable the new
-https://progress.opensuse.org/workflow/factory-proposal.html[factory
-workflow] to work with staging projects.  This plugin implements some
```

Comment:
Meanwhile the staging handling is part of OBS, which makes some more parts of this file also obsolete